src/control/property.rs

```diff
@@ -186,7 +186,7 @@ impl<'a> From<Value<'a>> for RawValue {
             Value::Boolean(false) => 0,
             Value::UnsignedRange(x) => x,
             Value::SignedRange(x) => x as u64,
-            Value::Enum(val) => val.value(),
+            Value::Enum(val) => val.map_or(0, EnumValue::value),
```

Comment:
Now again not sure about this one :grimacing:

Reply:
At this point we would actually need an EnumValue::Unknown to handle the back-and-forth conversion. But I think this is good enough at this point.